style.css

```diff
@@ -0,0 +1,53 @@
+body{
```

Comment:
please, follow the linters.yml instructions to make required changes, if you are confused, look at the linters.yml report the numbering by your left side begins this way 1. 3: 5
where 1 indicates the serial number, 3 indicate the line number of your code, and 5 indicates the points where the change is required on code line 3 number 1 error
![Screenshot (1725)](https://user-images.githubusercontent.com/31141344/220060946-dfa9a2d6-6b10-48f9-aab8-7fa0fdf40f3d.png)